components/Navbar.tsx

```diff
@@ -50,7 +52,7 @@ const Navbar = () => {
 
       {/* Mobile Nav */}
 
-      <section className="navsize:hidden flex  justify-between items-baseline pt-4 px-4 w-full  z-50">
+      <section className="navsize:hidden flex  justify-between items-baseline pt-4 pb-4 px-4 w-full  z-50">
```

Comment:
the text on mobile navbar only says Axiom Make it Axiom books and change its coulor to black plz